src/components/currency/Currency.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+const StyledCurrency = styled.span`
+  font-feature-settings: 'tnum' 1;
```

Comment:
Interesting! This is new to me

Reply:
I'm unsure if that is still necessary -> Copied from dashboard 😅